samples/23.a.hybrid-react-npm/README.md

```diff
@@ -0,0 +1,164 @@
+# Using different version of React for hosting app and Web Chat
+
+Starting from Web Chat 4.6, we are migrating to React Hooks and requiring React 16.8.6 or up.
+
+In this sample, we will show how to use Web Chat in an hosting app with React version 16.0.0, instead of the required 16.8.6.
```

Comment:
```suggestion
In this sample, we will show how to use Web Chat in a hosting app with React version 16.0.0, instead of the required 16.8.6.
```

Reply:
You need to teach me when to use "an" with words start with "h". Is it because "h" is not sound in "hour" so it sound like "Ours" so we need "an"?